File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let c=0;
    let maxi=0;
    for(let i=0;i<nums.length;i++){
        if(nums[i]==1){
            c++;
            maxi=Math.max(maxi,c);
        }else{
            c=0;
        }
    }
    return maxi;
};

File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cnt = maxi = 0
        for n in nums:
            if n==1 : 
               cnt+=1
               maxi=max(cnt,maxi) 
            else : cnt=0
        return maxi
     